[error tracking] Add Datadog Operator instructions for ET Backend standalone (#28483)
DataDog/documentation: content/en/error_tracking/backend/getting_started/single_step_instrumentation.md

- [Kubectl CLI][4] for installing the Datadog Agent.

{{< collapse-content title="Installing with Datadog Operator" level="h4" >}}
Follow these steps to enable Single Step Instrumentation across your entire cluster using the Datadog Operator. This enables tracing for all applications written in supported languages.

> To enable Single Step Instrumentation with the Datadog Operator:

Reviewer: ```suggestion
```

   spec:
     global:
       site: <DATADOG_SITE>
       tags:
         - env:<AGENT_ENV>
       credentials:
         apiSecret:
           secretName: datadog-secret
           keyName: api-key
       env:
         - name: DD_CORE_AGENT_ENABLED
           value: "false"
     features:
       apm:
         enabled: true
         errorTrackingStandalone:
           enabled: true
         instrumentation:
           enabled: true
           libVersions:
             java: "1"
             dotnet: "3"
             python: "2"
             js: "5"
             php: "1"
   ```
   Replace `<DATADOG_SITE>` with your [Datadog site][6] and `<AGENT_ENV>` with the environment your Agent is installed on (for example, `env:staging`).
4. Run the following command:
   ```shell
   kubectl apply -f /path/to/your/datadog-agent.yaml
   ```
5. Wait a few minutes for the Datadog Cluster Agent changes to apply, then restart your applications.
{{< /collapse-content >}}

{{< collapse-content title="Installing with Helm" level="h4" >}}
Follow these steps to enable Single Step Instrumentation across your entire cluster using Helm. This enables tracing for all applications written in supported languages.

> To enable Single Step Instrumentation with Helm:

Reviewer: ```suggestion
```

- [Kubectl CLI][4] for installing the Datadog Agent.

{{< collapse-content title="Installing with Datadog Operator" level="h4" >}}

> Follow these steps to enable Single Step Instrumentation across your entire cluster with the Datadog Operator. This automatically sends traces for all applications in the cluster that are written in supported languages.
> 
> To enable Single Step Instrumentation with the Datadog Operator:

Reviewer: ```suggestion
Follow these steps to enable Single Step Instrumentation across your entire cluster using the Datadog Operator. This enables tracing for all applications written in supported languages.
```

Author: Just to be sure: are you aware this is a copy-paste from the [Helm instructions](https://github.com/DataDog/documentation/blob/master/content/en/error_tracking/backend/getting_started/single_step_instrumentation.md?plain=1#L68-L70) that are a few lines below this in the same page? If so, should I also change the Helm instructions to be consistent?

   ```shell
   kubectl create secret generic datadog-secret --from-literal api-key=<YOUR_DD_API_KEY>
   ```
3. Create `datadog-agent.yaml` with the spec of your Datadog Agent deployment configuration. The simplest configuration is as follows:
   ```yaml
   apiVersion: datadoghq.com/v2alpha1
   kind: DatadogAgent
   metadata:
     name: datadog
   spec:
     global:
       site: <DATADOG_SITE>
       tags:
         - env:<AGENT_ENV>
       credentials:
         apiSecret:
           secretName: datadog-secret
           keyName: api-key
       env:
         - name: DD_CORE_AGENT_ENABLED
           value: "false"
     features:
       apm:
         enabled: true
         errorTrackingStandalone:
           enabled: true
         instrumentation:
           enabled: true
           libVersions:
             java: "1"
             dotnet: "3"
             python: "2"
             js: "5"
             php: "1"
   ```
   Replace `<DATADOG_SITE>` with your [Datadog site][6] and `<AGENT_ENV>` with the environment your Agent is installed on (for example, `env:staging`).
4. Run the following command:
   ```shell
   kubectl apply -f /path/to/your/datadog-agent.yaml
   ```

> 5. After waiting a few minutes for the Datadog Cluster Agent changes to apply, restart your applications.

Reviewer: ```suggestion
5. Wait a few minutes for the Datadog Cluster Agent changes to apply, then restart your applications.
```

Author: (same question as above)
